### DISCUSSION TITLE
How to return to grid view?

### DISCUSSION THREAD

**Discussion by merano2** [AlUlkesh/stable-diffusion-webui-images-browser#242] [Q&A] [answered]
If I select a picture, I cannot return to the grid/overview. Is there a keyboard shortcut or could you add a button for this plz?

**Comment by AlUlkesh** [comment, marked as answer]
There is a button in the top right corner:

![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/99896447/14047061-157a-4611-bf3a-04acfef7ba85)

**Comment by merano2** [comment]
oh, did not see this, thx!
